docs/README.md

```diff
@@ -0,0 +1,62 @@
+# Open Source Programs Web Docs
+
+This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
+
+## Installation
+
+```console
```

Comment:
@Rahulm2310  I guess there is no use of the word console here you can simply use ``` without console. And wherever mentioned